.gitignore

```diff
@@ -0,0 +1,44 @@
+### OSX specific things
+.DS_Store
+.AppleDouble
+.LSOverride
+
+# Icon must end with two \r
```

Comment:
To be honest, I grabbed this .gitignore setup from stack overflow, while trying to get all the right XCode junk excluded.